scripts/evaluate_samples.py

```diff
@@ -33,6 +35,9 @@ def warn(msg: Any):
 
 
 SAMPLES_DIR = mentat_dir_path / "samples"
+os.makedirs(SAMPLES_DIR, exist_ok=True)
```

Comment:
path objects have a `mkdir` which may be better to use here for consistency with the rest of the codebase.